int16 num # number of strawberry
sensor_msgs/Image img # corresponding frame
bbox[] bbox
